internal/store/digest.go

```diff
@@ -0,0 +1,38 @@
+package store
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"regexp"
+
+	"github.com/docker/distribution/reference"
+	"github.com/opencontainers/go-digest"
+)
+
+// computeDigest takes a bundle and produce a unigue reference.Digested
+func computeDigest(bundle io.WriterTo) (digest.Digest, error) {
+	b := bytes.Buffer{}
+	_, err := bundle.WriteTo(&b)
+	if err != nil {
+		return "", err
+	}
+	return digest.SHA256.FromBytes(b.Bytes()), nil
+}
```

Comment:
`computeDigest` (or a wrapper) should be available outside the store package in order ro recompute ID